arangod/RestServer/DatabaseFeature.cpp

```diff
@@ -675,6 +704,15 @@ int DatabaseFeature::dropDatabase(std::string const& name, bool waitForDeletion,
     delete oldLists;
 
     TRI_ASSERT(!vocbase->isSystem());
+
+#if USE_IRESEARCH
```

Comment:
@ObiWahn let's remove USE_IRESEARCH since views are generic ArangoDB concept, e.g. there is no `ifdef` around `ClusterInfo::getViews`